include/deal.II/differentiation/ad/ad_helpers.h

```diff
@@ -2934,6 +2934,347 @@ namespace Differentiation
     }; // class ADHelperPointLevelFunctionsBase
 
 
+
+    /**
+     * A helper class that facilitates the evaluation of a scalar function,
+     * its first and second derivatives (gradient and Hessian). This class
```

Comment:
Well, I would argue that since first and second are adjectives that there is no ambiguity here. But I've rephrased it now so its quite clear what I mean :-)

Reply:
> https://www.usatoday.com/story/money/2018/02/08/oxford-comma-lawsuit/320944002/

That's one expensive grammatical error :-D

Reply:
And it's the company paying for the comma-carelessness of the state legislature :)

Not mentioned is that the state legislature of the state passed one mammoth bill a couple of months ago adding commas in all places where an Oxford comma should go. I thought that was interesting, too, that someone took the time to read through all state laws and mark up commas :-)

Either way, it doesn't matter in the current context as you mention.